libr/bin/p/bin_pe64.c

```diff
@@ -1,7 +1,6 @@
 /* radare - LGPL - Copyright 2009-2015 - nibble, pancake */
-
 #define R_BIN_PE64 1
-#include "bin_pe.c"
+#include "bin_pe.inc"
```

Comment:
i did some tests and technically .inc is exactly the same as .c, it doesnt detects dependencies and it doesnt specify the language inside. so i dont see using .inc a good idea at all.

Reply:
I do see a difference instead. When compiling `master` (with meson+ninja at least) i get a:
```
[545/1115] Compiling C object 'libr/bin/libr@bin@@r_bin@sha/p_bin_pe64.c.o'.
In file included from ../libr/bin/p/bin_pe64.c:4:0:                                                                                                                                          
../libr/bin/p/bin_pe.c:35:15: warning: ‘load_buffer’ defined but not used [-Wunused-function]
 static void * load_buffer(RBinFile *bf, RBuffer *buf, ut64 loadaddr, Sdb *sdb) {                    
```
but I don't with @MaskRay 's PR.